docs/02-concepts/01-data-processing/01-pii-field-encryption.md

```diff
@@ -64,50 +68,29 @@ customer journeys, without compromising the privacy of your users.
 
 ## Using the decrypted data
 
-### Identify the consent levels you need.
+### Identify the data purposes you need
 
 Ask
 the [Data Protection Officer](https://edps.europa.eu/data-protection/data-protection/reference-library/data-protection-officer-dpo_en)
-the
-specific consent levels your use case requires. This will return a list
-of values [^3] that you need.
-
-### Consent level types
-
-STRM Privacy supports two types of consent levels when creating a
-decrypted output stream: granular and cumulative.
-
-1. _Cumulative_: Only the highest consent level is configured on the
-   output stream. All consent levels from zero (included) up to this
-   level (included) are decrypted in the output stream.
+the specific data purposes your use case requires, or is allowed to use.
+Data for purposes that do not apply to your use case will not be decrypted when creating a _privacy stream_.
 
-2. _Granular_: All the consent levels that are to be decrypted in the
-   output stream are explicitly configured. This way, it’s possible to
-   have "gaps" between the consent levels. For example, can specify
-   level 1 and 4, which means that all other levels, including 2 and 3
-   remain encrypted.
+### Create a privacy stream
 
-The resulting set of consent levels effects two things, see:
-[Create a decrypted stream](#create-a-decrypted-stream)
+Here you instruct STRM Privacy to create a derived stream where event data corresponding to the requested purposes is decrypted.
+This only happens for events where (data subject) consent is granted for these purposes. As such, STRM Privacy will:
 
-### Create a decrypted stream {#create-a-decrypted-stream}
+- Exclude all events that have not been allowed to be used for these requested purposes.
 
-Here you instruct STRM Privacy to decrypt event data with corresponding to the requested consent
-levels, but only for data of which the data subject has provided the required consent. STRM Privacy will:
+- Decrypt event data (fields/attributes) filed under the purposes you requested (in the event's [data contract]).
```

Comment:
Are these `[data contract]` notations correct? I'm seeing it twice in this file, and I believe only `[text](link)` and `[text][link ref]` are valid constructs in Markdown.

Reply:
This works too and was used in a few places, but later I switched to the `[text][link ref]` notation. I can modify them for consistency.